examples/tv-app/tv-common/src/AppTv.cpp

```diff
@@ -684,7 +724,7 @@ CHIP_ERROR AppTvInit()
     ContentAppPlatform::GetInstance().SetupAppPlatform();
     ContentAppPlatform::GetInstance().SetContentAppFactory(&gFactory);
     gFactory.InstallContentApp((uint16_t) 1, (uint16_t) 11);
-    gFactory.InstallContentApp((uint16_t) 65521, (uint16_t) 32768);
+    gFactory.InstallContentApp((uint16_t) 65521, (uint16_t) 32769);
```

Comment:
could we name these constants here? what is 1, 11, 65521, 32769 and such and why is 32769 better than 32768?

Reply:
created variables that describe what's going on.